PHP/AutoComplete/Getting-Started.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started
+title: Getting Started with PHP AutoComplete Control | Syncfusion
+description: Learn here about getting started with Syncfusion Essential PHP AutoComplete Control, its elements, and more.
 platform: php
 control: Autocomplete 
 documentation: ug
 ---
 
-#Overview
+# Overview of PHP AutoComplete
 
 The PHP AutoComplete control is a textbox control that provides a list of suggestions based on your query. When you enter text into the text box, the control performs a search operation and provides a list of results. There are several filter types available to perform the search.
 
-# Getting Started
```

Comment:
why Overview & Getting started in same page?

Reply:
Removed Overview